components/network/ntp/ntp.p5m

```diff
@@ -19,7 +19,7 @@
 # CDDL HEADER END
 #
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
-# Copyright (c) 2023, Friedrich Kink. All rights reserved.
```

Comment:
Please never remove copyrights of others without their consent.

Reply:
changed back

Reply:
Where is Friedrich Kink?  What exactly you added back?

Reply:
have forgotten git add for ntp.p5m